contexts/audit/ruby/Makefile

```diff
@@ -0,0 +1,14 @@
+-include ../../prerequisites/tasks.mk
+-include ../../prerequisites/ruby.mk
+
+##>> run bundle audit
+
+.DEFAULT_GOAL := all
+.PHONY: all
+
+## all (default): run bundle audit
+all: bundle
+	@cd $(base_path) && bundle exec bundle audit --update
+
+ignore:
```

Comment:
Should this also depend on `bundle`?

Reply:
Yes, it would depend. I forgot to call. But now, as said in the previous comment, this target does not exist anymore.